ee_tests/src/page_objects/space_dashboard/add_to_space_dialog.ts

```diff
@@ -255,7 +255,6 @@ export class AddToSpaceDialog extends ui.ModalDialog {
     // await ghPasswd.sendKeys(browser.params.github.password);
     // await ghPasswd.submit();
     // END: Workaround for the Github login
-
     await launcher.selectGithubOrganization(browser.params.github.username);
     await launcher.ghRepositoryText.sendKeys(name);
```

Comment:
This line is probably wrong... it created a GitHub repo https://github.com/ppitonak/testmay31525352606236testmay31525352606236

Reply:
You are right - I removed that line and it should work OK.

Reply:
You are right, the field needs to be cleaned up before sending keys.